eco.Rmd

```diff
@@ -25,42 +25,47 @@ finally, point to books on ecological modeling especially emphasizing Alain Zuur
 
 Fog oases are one of the most fascinating vegetation formations I have ever encountered. 
 These formations, locally termed *lomas*, develop on mountains along the coastal deserts of Peru and Chile.
-The desert's extreme harsh conditions and its remoteness spawned a truly unique and endemic flora and fauna on these mountains.
```

Comment:
I've cut-down on adverbs to make it more concise.

Reply:
Thank you very much!